superset-frontend/src/SqlLab/components/EstimateQueryCostButton/index.tsx

```diff
@@ -54,20 +57,20 @@ const EstimateQueryCostButton = props => {
   );
 
   const onClick = () => {
```

Comment:
hmm, do we even need the `onClick` function if all it does is call `getEstimate`? Seems like we can put `getEstimate` directly on the button's onClick prop

Reply:
Done.